### PR TITLE
KAFKA-15483 Add KIP-938 and KIP-866 metrics to ops.html

### PR DESCRIPTION
**Pull request by mumrah**

### CODE REVIEW

File: 36/ops.html [hunk 1]
```diff
@@ -1980,6 +1980,28 @@ <h5 class="anchor-heading"><a id="kraft_quorum_monitoring" class="anchor-link"><
...
+  </tr>
+  <tr>
+    <td>Latest Metadata Snapshot Size</td>
+    <td>The total size in bytes of the latest snapshot that the node has generated. If none have been generated yet, this is the size
+    of the latest snapshot that was loaded. If no snapshots have been generated or loaded, this is 0.</td>
+    <td>kafka.server:type=SnapshotEmitter,name=LatestSnapshotGeneratedBytes</td>
+  </tr>
+  <tr>
+    <td>Latest Metadata Snapshot Age</td>
+    <td>The interval in miliseconds since the latest snapshot that the node has generated.
```
Comment (satishd) [inline 36/ops.html]: nit typo: `miliseconds` -> `milliseconds`

**Review by satishd — COMMENTED** [review]
Thanks @mumrah for the PR, left a minor comment.

**Comment by satishd** [comment]
@mumrah  I do not see the documentation in the latest kafka-site docs for the below metrics that are part of KIP-866, correct me if I missed them in the existing docs. 
```
ZkWriteBehindLag
ZkWriteSnapshotTimeMs
ZkWriteDeltaTimeMs
```

**Comment by mumrah** [comment]
Thanks for calling those out, @satishd. I've added them as well.

**Review by rondagostino — APPROVED** [review]
LGTM

**Review by satishd — APPROVED** [review]
Thanks @mumrah for addressing the review comments. LGTM.

**Pull request merged**